src/components/AdaptiveCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
We need to move this component to `components/default` folder as per our project structure.